api-helpers/mailFunctions.ts

```diff
@@ -0,0 +1,51 @@
+import type { Blog } from '@prisma/client';
+import { UserRole } from '@prisma/client';
+import mailgun from 'mailgun-js';
+
+import { closeConnection, openConnection } from './db';
+
+const mg = mailgun({ apiKey: process.env.MAILGUN_API_KEY!, domain: process.env.MAILGUN_DOMAIN! });
+
+const sendEmail = async (recepients: readonly string[], subject: string, text: string) => {
+  const data = {
+    from: process.env.MAILGUN_FROM!,
+    to: recepients.join(', '),
```

Comment:
A może jak ktoś poda maila w formularzu to damy tutaj Reply-To na ten email?
A może w ogóle zrobić email w formularzu obowiązkowy?

Reply:
Jestem za.